build-scripts/build_course_zips.js

```diff
@@ -45,108 +43,115 @@ const distPath = options.dist
 const coursesPath = options.courses
 const zipsPath = options.zips
 
-// clear out the distribution path and run the webpack build
-rimraf.sync(distPath)
-if (shell.exec("npm run build:webpack").code !== 0) {
-  console.error("Webpack build failed")
-  process.exit(1)
+
+// Walk a tree and produce { root, relPath, file } for each file found
+async function* iterateTree(rootPath, relPath = ".") {
```

Comment:
In general we should use async variants of file system operations where we can so that they aren't blocking other JS work. I'm also a little skeptical of third party npm libraries which aren't well used, and walk-sync only appears to have 58 stars on github.

Reply:
Okay, that makes sense.  We should still put things like this in `helpers.js` though I think.

Reply:
Sure, it's been moved and a test is added for it